docs/protocol/specifications/vm1.md

```diff
@@ -1260,9 +1296,13 @@ Fails if executed in the [transaction context](#transaction-context).
 
 Code  | Stack Diagram   | Cost
 ------|-----------------|-----------------------------------------------------
-0x50, 0x61, 0x62, 0x65, 0x66, 0x67, 0x68, 0x8a, 0x8d, 0x8e, 0xa6, 0xa7, 0xa9, 0xab, 0xb0..0xbf, 0xca, 0xcd..0xcf, 0xd0..0xff  | (∅ → ∅)     | 1
+0x50, 0x61, 0x62, 0x65, 0x66, 0x67, 0x68, 0x8a, 0x8d, 0x8e, 0xa6, 0xa7, 0xa9, 0xab, 0xb0..0xbf, 0xcd..0xcf, 0xd0..0xff  | (∅ → ∅)     | 1
+
+The unassigned codes are reserved for future expansion.
+
+If the [expansion flag](#vm-state) is set, these opcodes have no effect on the state of the VM apart from reducing run limit by 1.
```

Comment:
> these opcodes have no effect on the state of the VM apart from reducing run limit by 1

"...when executed."

Also, it's technically wrong to say they have no other effect on the state of the VM: they also increase the PC by one. Better to say, "these opcodes act exactly like `NOP`." (Except that I see there no longer is a `NOP` in the spec. Since when? The implementation still includes `NOP` at 0x61.)